docs/admin/postgres.mdx

```diff
@@ -6,7 +6,9 @@ Sourcegraph uses several PostgreSQL databases to support various functionality.
 - `codeintel-db`: provides support for lsif data and part of the code-intelligence
 - `codeinsights-db`: provides support for code insights data
 
-## Version requirements
+## Requirements
+
+### Version requirements
 
 We support Postgres 16 and above. Older versions of Sourcegraph supported 12 see our [Postgres 12 deprecation notice](/admin/postgres12_end_of_life_notice) for more details. Starting in Sourcegraph 6.0.0, sourcegraph services will no longer connect to a PostgreSQL 12 or lower database. Instead displaying an error message in container logs for services attempting to connect to a PostgreSQL database below version 16.
```

Comment:
```suggestion
We support Postgres 16 and above. Older versions of Sourcegraph supported 12 see our [Postgres 12 deprecation notice](/admin/postgres12_end_of_life_notice) for more details. Starting in Sourcegraph 6.0.0, sourcegraph services will no longer connect to a PostgreSQL 12 or lower database. Instead, it will display an error message in container logs for services attempting to connect to a PostgreSQL database below version 16.
```